### PR TITLE
attempt at rendering entire samples

### PR DESCRIPTION
**Pull request by pelikhan**
I've been trying to render the entire sample on an image but I can't somehow relayout the page/reset buttons. any ideas?

### CODE REVIEW

**Comment by pelikhan** [comment]
@tballmsft  @eanders-ms

File: editor.ts [hunk 1]
```diff
@@ -270,12 +283,24 @@ namespace microcode {
...
+            this.layout()
+        }
+
+        private layout() {
+            this.scrollroot.xfrm.localPos = new Vec2(
+                Screen.LEFT_EDGE,
+                Screen.TOP_EDGE + TOOLBAR_HEIGHT + TOOLBAR_MARGIN
+            )
+            this.scrollDest.copyFrom(this.scrollroot.xfrm.localPos)
+            this.pageBtn.xfrm.localPos.x = Screen.RIGHT_EDGE - 32
```
Comment (eanders-ms) [inline editor.ts:302]: @pelikhan I was going to suggest this for relayout of the toolbar. Is it not working? You might try resetting hudroot's localPos, if Screen.TOP_EDGE changed. `this.hudroot.xfrm.localPos = new Vec2(0, Screen.TOP_EDGE)`

Comment (pelikhan) [reply editor.ts]: i went for extreme and spanning editors on each page/screen size

**Comment by tballmsft** [comment]
Looking good!

**Comment by pelikhan** [comment]
looks ok now 
image

**Pull request merged**